TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -164,5 +173,16 @@ class AllStopsTableViewController: UITableViewController {
     @objc func backAction() {
         navigationController?.popViewController(animated: true)
     }
-
+    
+    // MARK: DZNEmptyDataSet
+    
+    func image(forEmptyDataSet scrollView: UIScrollView!) -> UIImage! {
+        return #imageLiteral(resourceName: "road")
```

Comment:
@mattbarker016 think Yana mentioned this to you at the meeting but think it would be good to have a custom icon for empty all stops

Reply:
Agreed, let's just use this one for now

Reply:
I changed the image to the big location pin -- can't remember if we decided to wait for a completely different image to merge or just go ahead with the pin?